curriculum/challenges/english/09-information-security/information-security-with-helmetjs/disable-dns-prefetching-with-helmet.dnsprefetchcontrol.english.md

```diff
@@ -9,7 +9,7 @@ forumTopicId: 301577
 ## Description
 <section id='description'>
 As a reminder, this project is being built upon the following starter project on <a href='https://glitch.com/edit/#!/remix/clone-from-repo?REPO_URL=https://github.com/freeCodeCamp/boilerplate-infosec/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-infosec/'>GitHub</a>.
-To improve performance, most browsers prefetch DNS records for the links in a page. In that way the destination ip is already known when the user clicks on a link. This may lead to over-use of the DNS service (if you own a big website, visited by millions people…), privacy issues (one eavesdropper could infer that you are on a certain page), or page statistics alteration (some links may appear visited even if they are not). If you have high security needs you can disable DNS prefetching, at the cost of a performance penalty.
+To improve performance, most browsers prefetch DNS records for the links in a page. In that way the destination IP is already known when the user clicks on a link. This may lead to over-use of the DNS service (if you own a big website, visited by millions of people…), privacy issues (one eavesdropper could infer that you are on a certain page), or page statistics alteration (some links may appear visited even if they are not). If you have high security needs you can disable DNS prefetching, at the cost of a performance penalty.
```

Comment:
Making @RandellDawson's suggestions easier to implement with the suggestion feature, along with some modifications of my own (more direct writing, changing nouns to verbs, typos).

```suggestion
To improve performance, most browsers prefetch DNS records for the links in a page. In that way, the destination IP is already known when the user clicks on a link. This may lead to overuse of the DNS service (if you own a big website, visited by millions of people…), privacy issues (one eavesdropper could infer that you are on a certain page), or altering page statistics (some links may appear visited even if they are not). If you have high security needs, you can disable DNS prefetching but at the cost of lower performance.
```